common/types/status.go

```diff
@@ -4,18 +4,33 @@ package types
 // Status represents the stage that a swap is at.
 type Status byte
 
+// Status values
 const (
-	ExpectingKeys Status = iota //nolint:revive
+	// ExpectingKeys is the status of the taker between taking an offer and
+	// receiving a response with swap keys from the maker. It is also the
+	// maker's status after creating an offer up until receiving keys from a
+	// taker accepting the offer.
+	ExpectingKeys Status = iota
+	// KeysExchanged is the status of the maker after a taker accepts his offer.
 	KeysExchanged
+	// ETHLocked is the taker status after locking her ETH up until confirming
+	// that the maker locked his XMR.
 	ETHLocked
+	// XMRLocked is the maker's state after locking the XMR up until he confirms
+	// that the the taker has set the contract to ready.
 	XMRLocked
+	// ContractReady is the taker's state after verifying the locked XMR and
+	// setting the contract to ready.
 	ContractReady
 	// CompletedSuccess represents a successful swap.
 	CompletedSuccess
 	// CompletedRefund represents a swap that was refunded.
 	CompletedRefund
-	// CompletedAbort represents the case where the swap aborts before any funds are locked.
+	// CompletedAbort represents the case where the swap aborts before any funds
+	// are locked.
 	CompletedAbort
+	// UnknownStatus is a placeholder for unmatched status strings.
+	// TODO: Why is it an indicator that a swap is ongoing?
```

Comment:
I was trying to understand this line when I wrote that TODO:
https://github.com/AthanorLabs/atomic-swap/blob/master/common/types/status.go#L100

Reply:
I wanted that function to only return false if the status was explicitly a completed one, but I think that `UnknownStatus` case should never be hit. we can change to a panic/err instead?

Reply:
Done: https://github.com/AthanorLabs/atomic-swap/commit/92d0ad25cdc0788dcfdabe968861dd1ff909e6e5